packages/utils/lib/environment/detection.ts

```diff
@@ -3,12 +3,13 @@ import { NodeEnv, localhostUrl } from './constants.js';
 export const baseUrl = process.env['NANGO_SERVER_URL'] || localhostUrl;
 export const basePublicUrl = process.env['NANGO_PUBLIC_SERVER_URL'] || baseUrl;
 
+export const isDocker = process.env['SERVER_RUN_MODE'] === 'DOCKERIZED';
 export const isStaging = process.env['NODE_ENV'] === NodeEnv.Staging;
 export const isProd = process.env['NODE_ENV'] === NodeEnv.Prod;
 export const isCloud = process.env['NANGO_CLOUD']?.toLowerCase() === 'true';
-export const isLocal = baseUrl === localhostUrl;
 export const isEnterprise = process.env['NANGO_ENTERPRISE']?.toLowerCase() === 'true';
-export const isDev = process.env['NODE_ENV'] === NodeEnv.Dev;
+export const isDev = !isDocker && process.env['NODE_ENV'] === NodeEnv.Dev;
+export const isLocal = !isCloud && !isEnterprise && !isDocker && isDev;
```

Comment:
hard to see if those changes have any side effect. Those conditions are starting to be quite complicated

Reply:
yep, that's why I would love for us to use feature flags instead

Reply:
I don't understand the conditions fully tbh. `isLocal`, `isDev`. I think it might be worth going to feature flags now even instead of this convoluted logic

Reply:
I copied isDocker to isDev just in case, but actually I'll remove this constant since it's not used in the code.
There is not an easy way to be sure we are in a local install so it's just "not everything" :D